include/trompeloeil.hpp

```diff
@@ -3498,6 +3991,9 @@ namespace trompeloeil
     unsigned long trompeloeil_expectation_line;                                \
     const char *trompeloeil_expectation_string;                                \
                                                                                \
+    using call_params_type_t = ::trompeloeil::call_params_type_t<sig>;         \
+    using return_of_t = ::trompeloeil::return_of_t<sig>;                       \
```

Comment:
These types really should have a `trompeloeil_` prefix to their names. It's not unimaginable that anyone may want to mock a function named `return_of_t`, or `call_params_type_t`.

Reply:
Done.